### DISCUSSION TITLE
Always trigger validation after the change in any field

### DISCUSSION THREAD

**Discussion by anhquande** [react-hook-form/react-hook-form#3366] [Q&A]
Hello,

In my form, i have to trigger a validation after the change in any field. 
According to the documentation (https://react-hook-form.com/api#useForm), i can set 'reValidateMode'  to 'onChange'. But it triggers only the validation of the field where the change just happens. I want to trigger the validation of all other fields as well. 

How can i do it ?

Thanks

**Comment by Cold-A-Muse** [comment]
Did you try form.trigger() ?

**Comment by bluebill1049** [comment]
rhf is focused on field-level validation, if you need form level, then you will have to use `trigger`.
